src/connector/src/sink/big_query.rs

```diff
@@ -536,6 +536,11 @@ impl Sink for BigQuerySink {
         self.check_column_name_and_type(big_query_schema)?;
         Ok(())
     }
+
+    fn update_config(&mut self, config: BTreeMap<String, String>) -> Result<()> {
+        self.config = BigQueryConfig::from_btreemap(config)?;
+        Ok(())
+    }
```

Comment:
I don't think update partial props on Compute Node is a good practice, we should not rely on the state kept by CN. 
A preferred way is to do the updates on meta and dispatch a full connector props to CN.